### PR TITLE
[New Model]: Secondary Material Content (split into shared, calculated, verifiable)

### LINKED ISSUE #434: [Model Update]: Secondary Material Content
## Update Reason
The SMC working group would like to update the existing model to get closer to existing Initiatives like WBCSD, include information about "Bruttomaterialeinsatzgewicht" and enable a more accurate definition of what post-consumer material is coming from EoL Vehicles and what not. The existing base structure of SMC will remain (Part -> List of Materials -> Material -> SMC Information).

## MS1 Criteria

- [x] The model that should be updated exists
- [x] The referenced use case exists
- [x] The potential updates are discussed with all stakeholders
- [x] A decision on versioning was made in consensus with the affected parties [according to the guidelines](https://github.com/eclipse-tractusx/sldt-semantic-models/blob/main/documentation/GOVERNANCE.md#versioning).
- [x] The potential update will be either 
  - [ ] backward compatible or 
  - [x] a (chain of) migration strategy(-ies) from all non-deprecated previous versions to the new model will be developed
- [x] A modeling steward has been assigned

### PR DESCRIPTION
**Pull request by catroest**
## Description
split of secondary material content aspect into calculated, verifiable and shared (as basis)

content changes:
-	Separate post consumer and post consumer automotive
-	Mass-balancing option for mechanical recycling
-	Add gross material input weight (Bruttomaterialeinsatzgewicht)

Closes #434 

## MS2 Criteria
(to be filled out by PR reviewer)
- [x] the model **validates** with the SAMM SDS SDK in the version specified in the Readme.md of this repository by the time of the MS2 check  (e.g., 'java -jar samm-cli.jar aspect \<path-to-aspect-model\> validate ). The  SAMM CLI is available [here](https://eclipse-esmf.github.io/esmf-developer-guide/tooling-guide/samm-cli.html) and in [GitHub](https://github.com/eclipse-esmf/esmf-sdk/releases/tag/v2.3.2)
- [x] use **Camel-Case** (e.g., "MyModelElement" or "TimeDifferenceGmtId", when in doubt follow https://google.github.io/styleguide/javaguide.html#s5.3-camel-case)
- [x] the identifiers for all model elements **start with a capital letter** except for properties
- [x] the identifier for **properties starts with a small letter**
- [x] all model elements **at least contain the fields "preferred name" and "description"** in English language. The description must be comprehensible. It is not required to write full sentences but style should be consistent over the whole model
- [x] Property and the referenced Characteristic should not have the same name
- [x] the versioning in the URN **follows semantic versioning**, where minor version bumps are backwards compatible and major version bumps are not backwards compatible. 
- [x] use **abbreviations only when necessary** and if these are sufficiently common
- [x] **avoid redundant prefixes in property names** (consider adding properties to an enclosing Entity or even adapt the namespace of the model elements, e.g., instead of having two properties `DismantlerId` and `DismantlerName` use an Entity `Dismantler` with the properties `name` and `id` or use a URN like `io.catenax.dismantler:0.0.1`)
- [x] fields `preferredName` and `description` are not the same
- [x] **`preferredName` should be human readable** and follow normal orthography (e.g., no camel case but normal word separation)
- [x] name of aspect is singular except if it only has one property which is a Collection, List or Set. In theses cases, the aspect name is plural.
- [x] units are referenced from the SAMM unit catalog whenever possible
- [x] **use constraints** to make known constraints from the use case explicit in the aspect model 
- [x] when relying on **external standards**, they are referenced through a **"see"** element
- [x] all properties with an [simple type](https://eclipse-esmf.github.io/samm-specification/2.0.0/datatypes.html) have an example value
- [x] metadata.json exists with status "release"
- [x] generated json schema validates against example json payload
- [x] file RELEASE_NOTES.md exists and contains entries for proposed model changes 
- [x] all contributors to this model are mentioned in copyright header of model file

## MS3 Criteria
(to be filled out by semantic modeling team before merge to main-branch)
- [x] All required reviewers have approved this PR (see reviewers section)
- [x] The new aspect (version) will be implemented by at least one data provider
- [x] The new aspect (version) will be consumed by at least one data consumer
- [x] There exists valid test data
- [x] In case of a new (incompatible) major version to an existing version, a migration strategy has been developed
- [x] The model has at least version '1.0.0'
- [x] If a previous model exists, model deprecation has been checked for previous model
- [x] The release date in the Release Note is set to the date of the MS3 approval

### CODE REVIEW

**Comment by catroest** [comment]
@agg3fe
- please check for MS2
- SecondaryMaterialContent was split into shared and two other models which reference it
- Versioning number ok?
- Deprecation of "old" SMC can be started @LuisRickert

File: io.catenax.shared.secondary_material_content/2.0.0/SecondaryMaterialContent.ttl [hunk 1]
```diff
@@ -0,0 +1,411 @@
...
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.0.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.0.0#> .
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.0.0#> .
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.0.0#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <urn:samm:io.catenax.shared.secondary_material_content:2.0.0#> .
+
+:SecondaryMaterialContentShared a samm:Aspect ;
```
Comment (agg3fe) [inline io.catenax.shared.secondary_material_content/2.0.0/SecondaryMaterialContent.ttl:31]: this identifier should match the model name. here name should be `SecondaryMaterialContent`

File: io.catenax.shared.secondary_material_content/2.0.0/SecondaryMaterialContent.ttl [hunk 2]
```diff
@@ -0,0 +1,411 @@
...
+
+:PercentageOfMaterialWeightUnit a samm:Unit ;
+   samm:preferredName "Percentage of Material Weight Unit"@en ;
+   samm:referenceUnit unit:percent ;
+   samm:symbol "%" .
+
+:mechanicalRecycling a samm:Property ;
+   samm:preferredName "Mechanical Recycling"@en ;
+   samm:description "Materials that have undergone mechanical recycling."@en ;
+   samm:characteristic :MechanicalRecycling .
```
Comment (agg3fe) [inline io.catenax.shared.secondary_material_content/2.0.0/SecondaryMaterialContent.ttl:385]: Property and the referenced Characteristic should not have the same name

File: io.catenax.shared.secondary_material_content/2.0.0/SecondaryMaterialContent.ttl [hunk 3]
```diff
@@ -0,0 +1,411 @@
...
+
+:chemicalRecycling a samm:Property ;
+   samm:preferredName "Chemical Recycling"@en ;
+   samm:description "Materials that have undergone chemical recycling."@en ;
+   samm:characteristic :ChemicalRecycling .
```
Comment (agg3fe) [inline io.catenax.shared.secondary_material_content/2.0.0/SecondaryMaterialContent.ttl:390]: Property and the referenced Characteristic should not have the same name. please check other places also.

File: io.catenax.shared.secondary_material_content/2.0.0/SecondaryMaterialContent.ttl [hunk 4]
```diff
@@ -0,0 +1,411 @@
...
+   samm:properties ( :mechanicalRecycling :chemicalRecycling ) .
+
+:PostConsumerAutomotiveEntity a samm:Entity ;
+   samm:preferredName "Post Consumer Automotive Entity"@en ;
+   samm:description "Material generated by households or by commercial, industrial and institutional facilities in their role as end-users of END OF LIFE VEHICLES which can no longer be used for its intended purpose. This includes returns of material from the distribution chain.\n\nThis attribute ONLY contains post consume material which originates from the automotive industry. All other material must be included in \"post consumer\"."@en ;
+   samm:properties ( :chemicalRecycling :mechanicalRecycling ) .
+
+:PercentageOfMaterialWeightUnit a samm:Unit ;
+   samm:preferredName "Percentage of Material Weight Unit"@en ;
```
Comment (agg3fe) [inline io.catenax.shared.secondary_material_content/2.0.0/SecondaryMaterialContent.ttl:378]: `description` missing

**Comment by agg3fe** [comment]
> @agg3fe
> 
> * please check for MS2
> * SecondaryMaterialContent was split into shared and two other models which reference it
> * Versioning number ok?
> * Deprecation of "old" SMC can be started @LuisRickert

I would say, for calculated and verifiable model, you can change the version to 1.0.0 instead 2.0.0 as these are the new introduced models.

File: io.catenax.secondary_material_content_calculated/2.0.0/SecondaryMaterialContentCalculated.ttl [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+#######################################################################
+
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.0.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.0.0#> .
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.0.0#> .
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.0.0#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <urn:samm:io.catenax.secondary_material_content_calculated:2.0.0#> .
```
Comment (agg3fe) [inline io.catenax.secondary_material_content_calculated/2.0.0/SecondaryMaterialContentCalculated.ttl:29]: please change the version to 1.0.0

File: io.catenax.secondary_material_content_verifiable/2.0.0/SecondaryMaterialContentVerifiable.ttl [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+#######################################################################
+
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.0.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.0.0#> .
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.0.0#> .
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.0.0#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <urn:samm:io.catenax.secondary_material_content_verifiable:2.0.0#> .
```
Comment (agg3fe) [inline io.catenax.secondary_material_content_verifiable/2.0.0/SecondaryMaterialContentVerifiable.ttl:29]: please change the version to 1.0.0

File: io.catenax.shared.secondary_material_content/2.0.0/SecondaryMaterialContent.ttl (hunk 4 above)
Comment (agg3fe) [reply io.catenax.shared.secondary_material_content/2.0.0/SecondaryMaterialContent.ttl:379]: Have you added the description?

**Comment by agg3fe** [comment]
please add missing description. After that ready for MS2.

File: io.catenax.shared.secondary_material_content/2.0.0/SecondaryMaterialContent.ttl (hunk 4 above)
Comment (catroest) [reply io.catenax.shared.secondary_material_content/2.0.0/SecondaryMaterialContent.ttl:379]: + why even define a own unit, just use the predefined percent?

**Review by agg3fe — APPROVED** [review]
LGTM

**Comment by tom-rm-meyer-ISST** [comment]
FYI: @catroest we set the version of the shared to 1.0.0 and updated the unit catalog references also to samm 2.1.0. Also we updated the MS3 approval date.

**Pull request merged**